### ISSUE TITLE
add dev example

### ISSUE THREAD

**Issue by menelike** [plandem/redux-react-grid-layout#1]
I like the idea behind this library. 
For further help a default example is missing. Ideally like `npm run dev` in `react-grid-layout`.

**Comment by plandem** [comment]
will try to do it today/tomorrow. never did any npm modules, only used :)

**Comment by menelike** [comment]
sounds great, I help where I can, new to npm myself :)

**Comment by bytewiz** [comment]
Did you manage to build this? because I am actually doing it for one project currently so can drop it here afterwards with and example? :)

**Comment by plandem** [comment]
nope,i decided don't spend time on module. Just integrated like here and that's all :)
